.grit/patterns/go/path_to_filepath.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Replace `path.Join()` ⇒ `filepath.Join()`
+---
+
+Utilize `filepath.Join(...)` instead of `path.Join(...)` as it accommodates OS-specific path separators, mitigating potential issues on systems like Windows that may employ different delimiters.
+
+### references
+
+- [path.join-considered-harmful](https://parsiya.net/blog/2019-03-09-path.join-considered-harmful/)
+- [path.go](https://go.dev/src/path/path.go?s=4034:4066#L145)
+
+tags: #fix #correctness
+
+```grit
+language go
+
+`path.Join($params)` => `filepath.Join($params)`
```

Comment:
It should also rewrite the imports. We probably want a set of import utilities for Go similar to what we have for Python: https://github.com/getgrit/stdlib/blob/2d013450d2b1ff1df16f7b0c90cf8b1fb1ea5e79/.grit/patterns/python/py_imports.grit

Reply:
Hi @morgante, add `ensure_import` function